### PR TITLE
Deploy NetworkFence CRD 

### PR DESCRIPTION
**Pull request by Yuggupta27**
Add missing files for the creation of NetworkFence custom resource definition.

Fixes: #50 

Signed-off-by: Yug Gupta <yuggupta27@gmail.com>

### CODE REVIEW

**Comment by Yuggupta27** [comment]
With this fix, the NetworkFence CRD is created with `make install`
```
[yuggupta@fedora kubernetes-csi-addons]$ make install 
/home/yuggupta/go/src/github.com/kubernetes-csi-addons/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
/home/yuggupta/go/src/github.com/kubernetes-csi-addons/bin/kustomize build config/crd | kubectl apply -f -
customresourcedefinition.apiextensions.k8s.io/csiaddonsnodes.csiaddons.openshift.io configured
customresourcedefinition.apiextensions.k8s.io/networkfences.csiaddons.openshift.io configured
customresourcedefinition.apiextensions.k8s.io/reclaimspacejobs.csiaddons.openshift.io configured
```

File: config/crd/kustomization.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@
 resources:
 - bases/csiaddons.openshift.io_csiaddonsnodes.yaml
 - bases/csiaddons.openshift.io_reclaimspacejobs.yaml
+- bases/csiaddons.openshift.io_networkfences.yaml
+
```
Comment (nixpanic) [inline config/crd/kustomization.yaml:8]: is this empty line needed? Not sure it could break the next `+kubebuilder` line

**Review by nixpanic — COMMENTED** [review]
It is fine to add these generated files as a single commit. There is no value in having separate commits for these details.

**Review by nixpanic — APPROVED** [review]

File: PROJECT [hunk 2]
```diff
@@ -22,4 +22,13 @@ resources:
...
   path: github.com/csi-addons/kubernetes-csi-addons/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: false
+  controller: true
+  domain: openshift.io
+  group: csiaddons
+  kind: NetworkFence
+  path: github.com/nwFence/api/v1alpha1
```
Comment (Madhu-1) [inline PROJECT:32]: This path is not correct isn't it?

Comment (Yuggupta27) [reply PROJECT]: Nice catch!! Updated now

**Review by Madhu-1 — APPROVED** [review]

**Review by nixpanic — APPROVED** [review]

**Pull request merged**